src/checkUsersTry.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+function checkUsersTry(usersTry, numberToGuess) {
+  if (isNaN(+usersTry) || usersTry.length !== 4) {
+    return 'The number must contain only 4 digits!';
+  }
+
+  if (new Set(usersTry).size !== 4) {
+    return 'The number must contain only unique digits!';
+  }
```

Comment:
The condition here doesn't suit the error message. You are testing length with a static number, that's confusing. The better way to do this is the next:
```suggestion
  if (new Set(usersTry).size !== usersTry.length) {
    return 'The number must contain only unique digits!';
  }
```